promscale/manage-data/index.md

```diff
@@ -0,0 +1,14 @@
+# Manage data in Promscale
+Promscale provides different methods to manage the data you store in Promscale
+and help you reduce disk space use and improve query performance.
```

Comment:
Promscale provides different methods to manage the data, reduce disk space, and improve query performance.